### ISSUE TITLE
Depreciated constants - form HA Core 2025.1

### ISSUE THREAD

**Issue by pgale** [gadget-man/iparcelboxHA#8]
Looks like these will change in HA Core 2025.1

```
Logger: homeassistant.components.lock
Source: helpers/deprecation.py:205
integration: Lock (documentation, issues)
First occurred: 08:13:14 (2 occurrences)
Last logged: 08:13:14

SUPPORT_OPEN was used from iparcelbox, this is a deprecated constant which will be removed in HA Core 2025.1. Use LockEntityFeature.OPEN instead, please report it to the author of the 'iparcelbox' custom integration


Logger: homeassistant.components.binary_sensor
Source: helpers/deprecation.py:205
integration: Binary Sensor (documentation, issues)
First occurred: 08:13:14 (8 occurrences)
Last logged: 08:13:14

DEVICE_CLASS_OPENING was used from iparcelbox, this is a deprecated constant which will be removed in HA Core 2025.1. Use BinarySensorDeviceClass.OPENING instead, please report it to the author of the 'iparcelbox' custom integration
DEVICE_CLASS_DOOR was used from iparcelbox, this is a deprecated constant which will be removed in HA Core 2025.1. Use BinarySensorDeviceClass.DOOR instead, please report it to the author of the 'iparcelbox' custom integration
DEVICE_CLASS_LOCK was used from iparcelbox, this is a deprecated constant which will be removed in HA Core 2025.1. Use BinarySensorDeviceClass.LOCK instead, please report it to the author of the 'iparcelbox' custom integration
DEVICE_CLASS_BATTERY_CHARGING was used from iparcelbox, this is a deprecated constant which will be removed in HA Core 2025.1. Use BinarySensorDeviceClass.BATTERY_CHARGING instead, please report it to the author of the 'iparcelbox' custom integration


Logger: homeassistant.const
Source: helpers/deprecation.py:205
First occurred: 08:13:04 (4 occurrences)
Last logged: 08:13:14

DEVICE_CLASS_BATTERY was used from iparcelbox, this is a deprecated constant which will be removed in HA Core 2025.1. Use SensorDeviceClass.BATTERY instead, please report it to the author of the 'iparcelbox' custom integration
DEVICE_CLASS_TIMESTAMP was used from iparcelbox, this is a deprecated constant which will be removed in HA Core 2025.1. Use SensorDeviceClass.TIMESTAMP instead, please report it to the author of the 'iparcelbox' custom integration
```

**Comment by pgale** [comment]
Are you aware of this one? Have some time to fix though I guess.

**Comment by gadget-man** [comment]
Yep draft update is done, just wanted to check there were no other issues from issue #7 that we needed to publish at the same time.

**Comment by pgale** [comment]
Ah great - thanks.

**Comment by jb123567** [comment]
Is this going to be implemented before 2025.1? The logs still include the warning 😃

**Comment by gadget-man** [comment]
I've now pushed v0.0.16 as pre-release. Please give it a try and let me know if you have any issues.
